cmake/modules/PrecompiledHeader.cmake

```diff
@@ -171,19 +171,24 @@ function(add_precompiled_header _target _input)
     set(_pch_flags_file "${_pch_binary_dir}/compile_flags.rsp")
     export_all_flags("${_pch_flags_file}")
     set(_compiler_FLAGS "@${_pch_flags_file}")
+    if(CMAKE_CXX_STANDARD EQUAL 11)
+        set(_cpp_standard_FLAG "-std=c++11")
```

Comment:
Oh, I see. Now need to figure out how to check value of CXX_STANDARD or just simply hard-code the correct flag.